src/views/Home.vue

```diff
@@ -0,0 +1,366 @@
+  },
+  methods: {
+    async getOrder() {
+      let resp;
+      try {
+        resp = await OrderService.getOrder(this.$route.params.orderId);
+        if (resp.status === 200 && !hasError(resp) && resp.data) {
+          this.order = resp.data;
+          console.log(this.order)
+          let productIds: any = new Set();
+          this.order.shipGroup.map((group: any) => {
+            group.selectedShipmentMethodTypeId = group.shipmentMethodTypeId;
+            group.items.map((item: any) => {
+              if (item.productId) productIds.add(item.productId);
+            })
+          })
+          this.productIds = [...productIds]
+          await this.fetchProducts(this.productIds)
+        } else {
+          showToast(translate("Order not Found"))
+        }
+      } catch (error) {
+        console.error(error)
+        showToast(translate("Something went wrong"))
+      }
+    },
+
+    async fetchProducts(productIds: any) {
+      const productIdFilter = productIds.reduce((filter: string, productId: any) => {
+        if (filter !== '') filter += ' OR '
+        return filter += productId;
+      }, '');
+
+      try {
+        const resp = await ProductService.fetchProducts({
+          "filters": ['productId: (' + productIdFilter + ')']
+        })
+
+        if (resp.status === 200 && !hasError(resp) && resp.data) {
+          resp.data.response.docs.forEach((product: any) => {
+            this.products[product.productId] = product
+          });
+        }
+      } catch (error) {
+        console.error(error)
+        showToast(translate("Something went wrong"))
+      }
+    },
+
+    getProduct(productId: string) {
+      return this.products[productId] ? this.products[productId] : {}
+    },
+
+    async saveShippingAddressChanges(shipGroup: any) {
+      let resp
+      const payload = {
+        "orderId": this.order.id,
+        "shipGroupSeqId": shipGroup.shipGroupSeqId,
+        "contactMechId": shipGroup.shipTo.postalAddress.id,
+        ...(shipGroup.selectedShipmentMethodTypeId === shipGroup.shipmentMethodTypeId) && { "isEdited": true },
+        "shipmentMethod": `${this.deliveryMethod}@_NA_`,
+        "contactMechPurposeTypeId": "SHIPPING_LOCATION",
+        "facilityId": "WH",
+        "toName": `${shipGroup.editedShipmentAddress.firstName} ${shipGroup.editedShipmentAddress.lastName}`,
+        "address1": shipGroup.editedShipmentAddress.address1,
+        "city": shipGroup.editedShipmentAddress.city,
+        "stateCode": shipGroup.editedShipmentAddress.stateCode,
+        "postalCode": shipGroup.editedShipmentAddress.postalCode,
+        "country": shipGroup.editedShipmentAddress.country
+      }
+
+      try {
+        resp = await OrderService.updateShippingAddress(payload);
+        if (resp.status === 200 && !hasError(resp) && resp.data) {
+          this.areChangesSaved = true;
+          showToast(translate("Changes saved"))
+        } else {
+          showToast(translate("Something went wrong"))
```

Comment:
Make the message more informative, like here we can say `Failed to update shipping addess`